File: app/src/main/java/com/example/msdc/api/CreditCrewResult.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

public class CreditCrewResult {
    @SerializedName("id")
    int id;

    @SerializedName("name")
    String name;

    @SerializedName("profile_path")
    String profilePath;

    @SerializedName("known_for_department")
    String jobDesk;

    @SerializedName("gender")
    int gender;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getProfilePath() {
        return profilePath;
    }

    public String getJobDesk() {
        return jobDesk;
    }

    public int getGender() {
        return gender;
    }
}

File: app/src/main/java/com/example/msdc/api/CreditCrewResult.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class CreditCrewResult (
    @field:SerializedName("id") val id : Int? = 0,
    @field:SerializedName("name") val name: String? = null,
    @field:SerializedName("profile_path") val profilePath: String? = null,
    @field:SerializedName("known_for_department") val jobDesk: String? = null,
    @field:SerializedName("gender") val gender : Int? = 0
) : Parcelable